frontend/src/composables/use-match-routes.ts

```diff
@@ -29,10 +30,10 @@ export const useMatchRoute = (
     return routes.includes(route.split("__")[0])
   }
 
-  const matches = ref(routeNameMatches(route.value?.name))
+  const matches = ref(routeNameMatches(String(route?.name)))
```

Comment:
Just curious, is `String()` needed for TypeScript, to actually convert it to a string, or something else? Why not `.toString()`? Or when is it not a string to begin with?

Reply:
Route now can be a string or a Symbol. And you're right, I'll replace this with `.toString()`.

Reply:
Added a `getRouteNameString` function, and replaced all occurrences of `String()` with `toString`.